src/Microsoft.DotNet.Darc/src/DarcLib/IRemote.cs

```diff
@@ -363,6 +363,14 @@ public interface IRemote
         /// <returns>Latest commit</returns>
         Task<string> GetLatestCommitAsync(string repoUri, string branch);
 
+        /// <summary>
+        ///     Get the commits in a repo on the specific branch 
+        /// </summary>
+        /// <param name="repoUri">Repository uri</param>
+        /// <param name="sha">Sha of the commit</param>
+        /// <returns>Return the commit matching the specified sha. Null if no commit were found.</returns>
+        public Task<Commit> GetCommitAsync(string repoUri, string sha);
```

Comment:
The "public" here is odd, Interfaces don't usually have access modifiers, and nothing else in this interface has it, so we shouldn't add it here.